### ISSUE TITLE
Blogs on the homepage should be curated

### ISSUE THREAD

**Issue by castrojo** [canonical/jujucharms.com#39]
We should be consuming 'juju' or some other rss tag from people's blogs so that things not related to Juju don't hit the front page, see screenshot.
![bug](https://cloud.githubusercontent.com/assets/1264109/5947257/985362f2-a709-11e4-9a2d-09fe148ad8d4.png)

**Comment by marcoceppi** [comment]
I vote the juju tag, it's what I've been using.

**Comment by Lucapaulina** [comment]
Sounds good. +1

**Comment by arosales** [comment]
Luca et all,

What is the method to get a blog curated here? Is there a certain list jujucharms.com looks at? If so, how do we update this list?

-thanks,
Antonio

**Comment by anthonydillon** [comment]
This feed is pulled from the Syndication on juju.ubuntu.com/blog. I do not permission to edit therefore will raise a ticket to have the Syndication updated.

**Comment by arosales** [comment]
We should also plan on how syndication can occur independent of juju.ubuntu.com

**Comment by castrojo** [comment]
@anthonydillon Who did you file a ticket with? We should sort this sooner rather than later.

**Comment by anthonydillon** [comment]
I got a reply to the RT stating:
"There's no way to do that, per se, as far as I can see.  What I ended up
doing was adding a juju tag to the blog and then changing the
syndication settings globally so that a post has to contain at least one
local tag to be syndicated.  I think that should do what you want."

This looks like it did not work. I'll chase up IS another solution as I don't have admin permission to the blog.
